system_metrics_collector/test/system_metrics_collector/test_utilities.cpp

```diff
@@ -86,6 +94,26 @@ TEST(UtilitiesTest, TestCalculateCpuActivePercentage)
   ASSERT_DOUBLE_EQ(test_constants::kCpuActiveProcSample_0_1, p);
 }
 
+TEST(UtilitiesTest, TestCalculatePidCpuActivePercentage)
```

Comment:
Is this test indicative of values we will see on real systems?

Reply:
Added some other tests that's more indicative of values on real systems.